polaris/hub/client.py

```diff
@@ -332,6 +333,33 @@ def get_dataset(self, owner: Union[str, HubOwner], name: str) -> Dataset:
 
         return Dataset(**response)
 
+    def read_zarr_file(self, owner: Union[str, HubOwner], name: str, path: str) -> zarr.hierarchy.Group:
+        """Read a Zarr file from a Polaris dataset
+
+        Args:
+            owner: Which Hub user or organization owns the artifact.
+            name: Name of the dataset.
+            path: Path to the Zarr file within the dataset.
+
+        Returns:
+            The Zarr object representing the dataset.
+        """
+        polaris_fs = PolarisFSFileSystem(
+            polaris_client=self,
+            dataset_owner=owner,
+            dataset_name=name,
+        )
+
+        try:
+            store = zarr.storage.FSStore(path, fs=polaris_fs)
+            return zarr.open(store, mode="r")
+        except TimeoutError as timeout_error:
+            raise PolarisHubError(
+                f"Timeout error: {timeout_error}. Consider increasing the expiration_seconds parameter and try again."
+            )
```

Comment:
`expiration_seconds` is not a parameter the user can actually influence as of now! 

I would do the following: 
1. Add a `default_timeout` parameter to `PolarisFSFileSystem`
2. Keep the `timeout` parameter for `cat_file` and `ls`, but its default value is `None`. 
3. If `None` in `cat_file` or `ls`, the timeout will be set to the `default_timeout`
4. Add a `timeout` parameter to the `read_zarr` method that is passed to `default_timeout` parameter when creating the PolarisFS object.

Then you can change this line to: 

```suggestion
            raise PolarisHubError(
                f"Timeout error. Consider increasing the `timeout` parameter and try again."
            ) from timeout_error
```